packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -118,7 +118,7 @@ function run (ipc, file, projectRoot) {
 
       return (await bundleRequire({ filepath: file })).mod
     } catch (err) {
-      if (err.stack.includes(`Cannot find package 'esbuild'`)) {
+      if (err.stack.includes(`Cannot find package 'esbuild'`) || err.stack.includes(`Cannot find module 'esbuild'`)) {
```

Comment:
I can't actually find an instance where "Cannot find package" is the error message -- at first I was worried that specific node versions might have changed the language of the stack, but I think I just made a typo in the first place.


```suggestion
      if (err.stack.includes(`Cannot find module 'esbuild'`)) {
```